packages/flutter/test/material/popup_menu_theme_test.dart

```diff
@@ -0,0 +1,295 @@
+      color: const Color(0xFFFFFFFF),
+      shape: RoundedRectangleBorder(borderRadius: BorderRadius.circular(2.0)),
+      elevation: 2.0,
+      textStyle: const TextStyle(color: Color(0xffffffff)),
+    ).debugFillProperties(builder);
+
+    final List<String> description = builder.properties
+        .where((DiagnosticsNode node) => !node.isFiltered(DiagnosticLevel.info))
+        .map((DiagnosticsNode node) => node.toString())
+        .toList();
+
+    expect(description, <String>[
+      'color: Color(0xffffffff)',
+      'shape: RoundedRectangleBorder(BorderSide(Color(0xff000000), 0.0, BorderStyle.none), BorderRadius.circular(2.0))',
+      'elevation: 2.0',
+      'text style: TextStyle(inherit: true, color: Color(0xffffffff))'
+    ]);
+  });
+
+  testWidgets('Passing no PopupMenuThemeData returns defaults',
+          (WidgetTester tester) async {
+    final Key popupButtonKey = UniqueKey();
+    final Key popupButtonApp = UniqueKey();
+    final Key popupItemKey = UniqueKey();
+
+    await tester.pumpWidget(MaterialApp(
+      theme: ThemeData(),
+      key: popupButtonApp,
+      home: Material(
+        child: Column(
+          children: <Widget>[
+            PopupMenuButton<void>(
+              key: popupButtonKey,
+              itemBuilder: (BuildContext context) {
+                final List<PopupMenuEntry<Object>> list = <
+                    PopupMenuEntry<Object>>[
+                      PopupMenuItem<void>(
+                        key: popupItemKey,
+                        child: const Text(''),
+                      ),
+                ];
+                return list;
+                },
+            ),
+          ],
+        ),
+      ),
+    ));
+
+    await tester.tap(find.byKey(popupButtonKey));
+    await tester.pumpAndSettle();
+
+    final Material button = tester.widget<Material>(
+      find.descendant(
+        of: find.byKey(popupButtonApp),
+        matching: find.byType(Material),
+      ).last,
+    );
+    expect(button.color, null);
+    expect(button.shape, null);
+    expect(button.elevation, 8.0);
+
+    final AnimatedDefaultTextStyle text = tester.widget<AnimatedDefaultTextStyle>(
+         find.descendant(
+           of: find.byKey(popupItemKey),
+           matching: find.byType(AnimatedDefaultTextStyle),
+         ).last,
+    );
+    expect(text.style.fontFamily, 'Roboto');
+    expect(text.style.color, const Color(0xdd000000));
+  });
+
+  testWidgets('PopupMenuEntry uses values from PopupMenuThemeData',
+          (WidgetTester tester) async {
+    final PopupMenuThemeData popupMenuTheme = _popupMenuTheme();
+    final Key popupButtonKey = UniqueKey();
+    final Key popupButtonApp = UniqueKey();
+    final Key popupItemKey = UniqueKey();
+
+    await tester.pumpWidget(MaterialApp(
+      theme: ThemeData(popupMenuTheme: popupMenuTheme),
+      key: popupButtonApp,
+      home: Material(
+        child: Column(
+          children: <Widget>[
+            PopupMenuButton<void>(
```

Comment:
Remove `<void>`

Reply:
The analyzer says that we can't remove `<void>` because [type annotations must be specified](https://dart-lang.github.io/linter/lints/always_specify_types.html).